headers/types/ground_mode/live_actor.h

```diff
@@ -0,0 +1,94 @@
+#ifndef HEADERS_LIVE_ACTOR_GROUND_MODE_H_
```

Comment:
Per convention, please use `HEADERS_TYPES_GROUND_MODE_LIVE_ACTOR_H_` (based on the full path)